### PR TITLE
Minor cleanups while looking at something VoF related.

### PR DESCRIPTION
**Pull request by bangerth**
Really minor stuff.

Related to #835.

### CODE REVIEW

File: source/volume_of_fluid/assembler.cc [hunk 1]
```diff
@@ -293,9 +291,11 @@ namespace aspect
         }
     }
 
+
+
     template <int dim>
     void VolumeOfFluidAssembler<dim>::local_assemble_internal_face_volume_of_fluid_system (const VolumeOfFluidField<dim> &field,
-        bool update_from_old,
+        const bool update_from_old,
```
Comment (tjhei) [inline source/volume_of_fluid/assembler.cc]: please change the signature in the .h as well.

**Comment by bangerth** [comment]
I fixed the place @tjhei marked in the .h file.

**Comment by tjhei** [comment]
:+1:

**Pull request merged**